docs/source/heroku_redis.rst

```diff
@@ -0,0 +1,48 @@
+Installing Heroku and Redis
+===========================
+
+Install Heroku
+--------------
+
+To run experiments locally or on the internet, you will need the Heroku Command
+Line Interface installed, version 3.28.0 or better. A Heroku account is needed
```

Comment:
Maybe "A Heroku.com account is needed..." or, "An actual Heroku _account_ is needed...". I was a little confused about the distinction between the Heroku CLI and this other thing (the Heroku account).

Reply:
If you want to launch experiments on the internet
you will also need a Heroku.com account, but is not needed for local debugging.

Reply:
?